src/main/java/frc/robot/constants/WristConstants.java

```diff
@@ -40,40 +29,9 @@ public class WristConstants {
   /** */
   public static final double GEAR_RATIO = 6;
 
-  /** */
-  public static final ArmFeedforward FF_MODEL = new ArmFeedforward(0, 0, 0, 0);
-
-  /** */
-  public static final PIDController PID_CONTROLLER = new PIDController(0, 0, 0);
-
-  /** */
-  public static final TrapezoidProfile TRAPEZOID_PROFILE =
-      new TrapezoidProfile(new TrapezoidProfile.Constraints(1, 2));
-
-  /** Ramp Rate of the wrist System ID in volts per second */
-  public static final double SYS_ID_RAMP_RATE = 1;
-
-  /** Setp Voltage of the wrist System ID in volts */
-  public static final double SYS_ID_STEP_VALUE = 7;
-
-  /** Timeout of the wrist System ID in volts */
-  public static final double SYS_ID_TIMEOUT = 30;
-
   /** How many degrees the wrist can be off its goal position for it to be sufficient */
   public static final double ALLOWED_DEVIANCE = 6;
 
-  /** */
-  public static final Translation3d TRANSLATION_FROM_ROBOT = new Translation3d(-0.2, 0, 0.255);
```

Comment:
Don't you need this constant for sim? (and eventually optimal inverse kinematics solving)